numpy/_core/code_generators/ufunc_docstrings.py

```diff
@@ -2830,7 +2830,7 @@ def add_newdoc(place, name, doc):
       >>> # n is 7, k is 4, m is 3
 
     The matmul function implements the semantics of the ``@`` operator
-    introduced in Python 3.5 following :pep:`465`.
+    defined in :pep:`465`.
```

Comment:
I'm not sure about this one

Reply:
How important is the exact version of Python that introduced the `@` operator, especially nowadays in the current documentation of NumPy?